programs/cleos/httpc.cpp

```diff
@@ -258,6 +258,7 @@ namespace eosio { namespace client { namespace http {
       response_result = fc::json::from_string(re);
    } catch(...) {
       // re reported below if print_response requested
+      std::cerr << "Error happens, below is the message received from server:" << std::endl << re << std::endl;
```

Comment:
Seems like `print_response = true;` would be a better way to handle this if you want to always `cerr` the `re`.

Reply:
Ok